app/templates/views/roadmap.html

```diff
@@ -26,49 +26,73 @@ <h1 class="heading-large">Roadmap</h1>
   <p class="govuk-body">The roadmap is only a guide. It does not cover everything we do, and some things may change.</p>
   <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.support')}}">contact us</a> if you have any questions about the roadmap or suggestions for new features.</p>
 
-  <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working on</h2>
-
-<h3 class="heading-small" id="now">Now</h3>
+  <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working on now</h2>
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Make it easier for services to pick their own <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.branding_and_customisation')}}">email branding</a></li>
-    <li>Let services preview their branding before sending an email</li>
-    <li>Make <a class="govuk-link govuk-link--no-visited-state" href="/features/email#send-files">sending files by email</a> more secure</li>
+    <li>Let services upload their own logo for emails and letters</li>
+    <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">replace GOV.UK Paas</a> by migrating our database to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

Comment:
There are a few things technically wrong about this sentence I'm afraid.

We are doing two things right now

Start to migrate our database from one currently hosted in the GOV.UK PaaS to one hosted hosted using AWS RDS
Migrating one of our applications that is currently hosted in the GOV.UK PaaS to being hosted using AWS ECS

You might be able to use those two sentences as is (plus making them link to the relevant pages) or happy to look at a suggestion

Reply:
@idavidmcdonald how’s this?

```suggestion
    <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/?nc2=h_ql_prod_fs_rds">Amazon RDS</a></li>
    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

Reply:
All good but you can make the AWS link slightly simpler, it only needs point at `https://aws.amazon.com/rds/`, you've got some query string on the end of yours which isn't necessary.